### PR TITLE
Implemented Stop Graph Feature

### LINKED ISSUE #330: Missing Stop Feature in Memory Graph
**Actual Behaviour.**  
Missing Stop Feature in Memory Graph.

**Expected Behaviour**  
There should be a stop functionality for the graph being plotted.

**Steps to reproduce it**  
Go to Memory graph, import data.

**Additional context**  
> Not available

**Would you like to work on the issue?**
Yes, would be working.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #330 

**Changes**: Implemented Stop Graph Feature

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3328214/app-debug.zip)

### CODE REVIEW

File: app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java [hunk 1]
```diff
@@ -315,14 +318,19 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
             progress.setCancelable(true);
             progress.setTitle(R.string.program_info_label);
             progress.setMessage(R.string.program_info);
             AlertDialog infoDialog = progress.create();
             infoDialog.show();
 
+        } else if (id == R.id.stop_data) {
+            parsedData = null;
+            Toast.makeText(getContext(), "Stopped", Toast.LENGTH_SHORT).show();
+            initializeMemGraph(memGraph);
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java]: Stopping should display the data right? But it seems to clear the whole graph :thinking:

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Also can we remove the white bubbles in those lines?

File: app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java (hunk 1 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java]: Okay so are you saying to stop the graph in it's last/latest position without completely clearing it?

Comment (CloudyPadmal) [reply app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java]: Yes

Comment (jddeep) [reply app/src/main/java/io/neurolab/fragments/MemoryGraphFragment.java]: Cool. will do that then. I thought that would be more appropriate as a pause feature.

**Comment by jddeep** [comment]
@CloudyPadmal Changes made, PR body updated. Have a look please.

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**